json-rpc/types/src/views.rs

```diff
@@ -480,6 +491,13 @@ impl TryFrom<ContractEvent> for EventDataView {
             EventDataView::AdminTransaction {
                 committed_timestamp_secs: admin_transaction_event.committed_timestamp_secs(),
             }
+        } else if event.type_tag() == &TypeTag::Struct(DiemIdDomainEvent::struct_tag()) {
```

Comment:
We need a test cover this. Maybe better to remove it from this PR and add to next PR with the event key change